transforms/angle-brackets/transform.js

```diff
@@ -317,7 +317,7 @@ function shouldIgnoreMustacheStatement(fullName, config, invokableData) {
     let strName = `${name}`; // coerce boolean and number to string
     return (isHelper || !isComponent) && !strName.includes('.');
   } else {
-    return !(KNOWN_HELPERS.includes(name) && config.helpers.includes(name));
+    return KNOWN_HELPERS.includes(name) && config.helpers.includes(name);
```

Comment:
> seem to have some doubts why it was working previously with telemetry and suddenly failing to work without telemetry.

I'm guessing that the tests haven't exercised this `} else {` block for a while?

I do think the previous logic was wrong, but I'm not sure about your change. Based on `let isHelper = mergedHelpers.includes(name) || config.helpers.includes(name);` from above, I would expect `return KNOWN_HELPERS.includes(name) || config.helpers.includes(name);`

I'm also wondering if there's additional logic inside the `if (isTelemetryData) {` block that needs to be captured like whether or not it's a component and whether or not the name includes a `.`.

Take this w/ a grain of salt b/c I haven't looked at the code outside of this fn, but it almost seems to me that you could remove `isTelemetryData` the `if/else` statement entirely and just execute what's in `if` block b/c of `helpers || []` and `components || []`.

Reply:
Agree, I think this should probably be an `||`. 🤔

Reply:
Agreed, will change that